src/RollbarLogHandler.php

```diff
@@ -73,55 +74,11 @@ public function log($level, $message, array $context = [])
             return;
         }
 
-        $context = $this->addContext($context);
```

Comment:
In the spirit of Open Source, and how Rollbar is making many other packages available under MIT license, I'm guessing @ArturMoczulski is open to it ... but his is a fork from Jens' package and neither Jens nor Artur have specified a license.